pkgs/by-name/au/authentik/package.nix

```
@@ -0,0 +1,236 @@
+
+    npmInstallFlags = [ "--include=dev" ];
+  };
+
+  authentik-django = python3.pkgs.buildPythonPackage {
+    pname = "authentik-django";
+    inherit version src;
+    format = "pyproject";
+
+    patchPhase = ''
+      substituteInPlace authentik/root/settings.py \
+        --replace 'Path(__file__).absolute().parent.parent.parent' "\"$out\""
+      substituteInPlace authentik/lib/default.yml \
+        --replace '/blueprints' "$out/blueprints"
+    '';
+
+    nativeBuildInputs = [ python3.pkgs.poetry-core ];
+    propagatedBuildInputs = with python3.pkgs; [
+      argon2-cffi
+      celery
+      channels
+      channels-redis
+      colorama
+      dacite
+      daphne
+      deepmerge
+      defusedxml
+      django
+      django-filter
+      django-guardian
+      django-model-utils
+      django-prometheus
+      django-redis
+      djangorestframework
+      djangorestframework-guardian2
+      docker
+      drf-spectacular
+      duo-client
+      facebook-sdk
+      flower
+      geoip2
+      gunicorn
+      httptools
+      kubernetes
+      ldap3
+      lxml
+      opencontainers
+      packaging
+      paramiko
+      psycopg
+      pycryptodome
+      pydantic
+      pydantic-scim
+      pyjwt
+      pyyaml
+      requests-oauthlib
+      sentry-sdk
+      structlog
+      swagger-spec-validator
+      twilio
+      twisted
+      ua-parser
+      urllib3
+      uvicorn
+      uvloop
+      watchdog
+      webauthn
+      websockets
+      wsproto
+      xmlsec
+      zxcvbn
+      jsonpatch
+    ] ++ [
+      codespell
+    ];
+
+    postInstall = ''
+      mkdir -p $out/web $out/website
+      cp -r lifecycle manage.py $out/${python3.sitePackages}/
+      cp -r blueprints $out/
+      cp -r ${webui}/dist ${webui}/authentik $out/web/
+      cp -r ${website} $out/website/help
+    '';
+  };
+
+  proxy = buildGoModule {
+    pname = "authentik-proxy";
+    inherit version;
+
+    src = runCommand "authentik-proxy-source" {} ''
+      mkdir -p $out/web
+      mkdir -p $out/authentik
+
+      cp -r ${src}/cmd ${src}/internal $out/
+      cp -r ${src}/authentik/lib $out/authentik/
+      cp ${src}/web/static.go $out/web/
+
+      cp ${src}/web/robots.txt ${src}/web/security.txt $out/web/
+      cp ${src}/go.mod ${src}/go.sum $out/
+    '';
+
+    patchPhase = ''
+      substituteInPlace internal/gounicorn/gounicorn.go \
+        --replace './lifecycle' "${authentik-django}/${python3.sitePackages}/lifecycle"
+      substituteInPlace web/static.go \
+        --replace './web' "${authentik-django}/web"
+      substituteInPlace internal/web/static.go \
+        --replace './web' "${authentik-django}/web"
+    '';
+
+    CGO_ENABLED = 0;
+
+    vendorHash = "sha256-8F9emmQmbe7R+xtGrjV5ht0adGasU6WAvLa8Wxr+j8M=";
+
+    postInstall = ''
+      mv $out/bin/server $out/bin/authentik
+    '';
+
+    subPackages = [ "cmd/server" ];
+  };
+
+in stdenvNoCC.mkDerivation {
+  pname = "authentik";
+  inherit src version;
+
+  patchPhase = ''
```

```suggestion
  postPatch = ''
```
we don't want to break patches